subprojects/snapshots/src/test/groovy/org/gradle/internal/fingerprint/impl/PathNormalizationStrategyTest.groovy

```diff
@@ -92,7 +93,7 @@ class PathNormalizationStrategyTest extends Specification {
     }
 
     def "sensitivity NAME_ONLY"() {
-        def fingerprints = collectFingerprints(NameOnlyFingerprintingStrategy.INSTANCE)
+        def fingerprints = collectFingerprints(NameOnlyFingerprintingStrategy.FINGERPRINT_DIRECTORIES)
```

Comment:
I'm not completely sure, but to me `DEFAULT` sounds like a better name here, too.

Reply:
Changed.